microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -161,16 +161,33 @@ Google may collect some data associated with the Media Router extension. To unin
+Collections storage and feature availability depend on your Microsoft Edge Sign-in and Sync settings.
 
-You can delete templates stored on the device and clear the cache data. Go to `edge://settings/privacy`. In **Clear browsing data** next to **Clear browsing data now**, select the **Choose what to clear** button. Choose your desired time range and type of data, then select the **Clear now** button.  Another way to delete cached data, go to `edge://settings/clearBrowserData` select the desired time range and type of data, then select the **Clear now** button.
 
-<!-- Above, should pick one way. Check UI for privacy updates -->
+<!-- ------------------------------ -->
+#### Signed in with a personal account, and Sync enabled
 
-You can use the title of your collection to find relevant Pinterest Topic pages when searching in Microsoft Bing. Go to `edge://settings/privacy`. Turn on the **Show suggestions from Pinterest in Collections** setting. Microsoft Edge does not send data about your collections to Pinterest. You can remove the suggestions and stop searches for Pinterest Topic pages. Go to `edge://settings/privacy` and turn off the **Show suggestions from Pinterest in Collections** setting.
-Collections aren't available when using **InPrivate** or **Guest** modes.
+When you sign in to Microsoft Edge with a personal Microsoft Account (MSA) and enable Sync, your collections, saved items, notes, and images are automatically saved on Microsoft cloud for free. You can access your collections from Microsoft Edge on any device where you're signed in with Sync enabled. You can even view and manage your collections online in any browser by visiting the [Collections portal](https://www.bing.com/saves) at Bing.com. Collections also enhance your experience with a personalized interest feed, if your personalization settings support this feature.
+
+If you want to disable Collections for a signed-in profile using a personal account with Sync enabled, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Show Collections and follow content creators in Microsoft Edge**.
```

Comment:
@apryzgoda - instead of adding sentence per Suggestion:
"When using a Microsoft Account (MSA), collections aren't available when using InPrivate browsing."
added sentence higher above from Jason:
"You cannot access Collections from an InPrivate window while signed in with a personal account."

latest Files Changed view:
https://github.com/MicrosoftDocs/edge-developer/pull/2778/files

closing